UnitsNet/GeneratedCode/Quantities/ElectricInductance.NetFramework.g.cs

```diff
@@ -582,14 +582,18 @@ public ElectricInductance ToUnit(ElectricInductanceUnit unit)
         ///     This is typically the first step in converting from one unit to another.
         /// </summary>
         /// <returns>The value in the base unit representation.</returns>
-        private double AsBaseUnit()
+        internal ElectricInductance AsBaseUnit()
```

Comment:
`ToBaseUnit()` to be consistent with `ToUnit()`?

Reply:
This change seems like extra overhead to me, is it needed? If we later convert to `class`, doesn't this introduce an additional object allocation for every unit conversion? 

I think the order seems reverse to me.
- Instead of `double AsBaseNumericType()` calling the "heavier" `MyQuantity AsBaseUnit()` method, shouldn't rather `AsBaseUnit()` call `AsBaseNumericType()`? Or am I missing something?

While touching this code:
- Can we please rename `AsBaseNumericType()` to something more intuitive, maybe `GetValueInBaseUnit()`? I keep forgetting what it does and it's name is just confusing to me.
- Can we also use explicit types instead of `var` in places where it is not obvious what the type is, such as https://github.com/angularsen/UnitsNet/blob/b4a42777ba3f7d383c50a1c617c4c61fe875f99b/UnitsNet/GeneratedCode/Quantities/ElectricInductance.NetFramework.g.cs#L607-L608

Reply:
I think those are all fair enough!